docs/configuration/multiple-delivery-clients.md

```diff
@@ -1,36 +1,128 @@
+
+## Using the default .NET implementation of the IMultipleDeliveryClientFactory
+
+### Installing the NuGet packages
+
+Advanced registration scenarios are handled by the [Kontent.Ai.Delivery.Extensions.DependencyInjection](https://www.nuget.org/packages/Kontent.Ai.Delivery.Extensions.DependencyInjection) NuGet package. You'll need to install it first:
+
+```sh
+Install-Package Kontent.Ai.Delivery.Extensions.DependencyInjection
+```
+
+### Registering the factory
+
+The SDK provides extension methods upon the `IServiceCollection` that allow registering delivery client factory builder. The builder is used to configure the factory and register the delivery client instances.
+
+```csharp
+public class Startup
+{
+    // ...
+
+    public void ConfigureServices(IServiceCollection services)
+    {
+        services.AddMultipleDeliveryClientFactory
+        (
+            factoryBuilder => factoryBuilder
+                .AddDeliveryClient
+                (
+                    "projectA",
+                    deliveryOptionBuilder => deliveryOptionBuilder
+                        .WithProjectId("<A_PROJECT_ID>")
+                        .UseProductionApi()
+                        .Build()
+                    optionalClientSetup =>
+                        optionalClientSetup.WithTypeProvider(new ProjectAProvider())
+                )
+        );
+    }
+}
 ```
-PM> Install-Package Kontent.Ai.Delivery.Extensions.DependencyInjection
-PM> Install-Package Autofac.Extensions.DependencyInjection
+
+### Registering multiple type providers
+
+If you're accessing two completely different projects, chances are they have a different content model and therefore the generated models for content types will differ. Extend the Startup class as follows:
+
+```csharp
+public class Startup
+{
+    // ...
+
+    public void ConfigureServices(IServiceCollection services)
+    {
+        services.AddMultipleDeliveryClientFactory
+        (
+            factoryBuilder => factoryBuilder
+                .AddDeliveryClient
+                (
+                    "projectA",
+                    deliveryOptionBuilder => deliveryOptionBuilder
+                        .WithProjectId("<A_PROJECT_ID>")
+                        .UseProductionApi()
+                        .Build(),
+                    optionalClientSetup =>
+                        optionalClientSetup.WithTypeProvider(new ProjectAProvider())
+                )
+                .AddDeliveryClient(
+                    "projectB",
+                    deliveryOptionBuilder => deliveryOptionBuilder
+                        .WithProjectId("<B_PROJECT_ID>")
+                        .UseProductionApi()
+                        .Build(),
+                    optionalClientSetup =>
+                        optionalClientSetup.WithTypeProvider(new ProjectBProvider())
+                )
```

Comment:
It seems that
`.Build()`
is missed here

Reply:
ae78d43